vnext/JSI/Shared/RuntimeHolder.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <jsi/jsi.h>
+#include <memory>
+
+namespace facebook {
+namespace jsi {
+
+// An instance of this interface is expected to
```

Comment:
>// [](start = 0, length = 2)

(nit) We should have proper documetation standard at least for new code, eg, doxygen. So comments would be:
//!